src/commands/slowlog-get.json

```diff
@@ -6,7 +6,7 @@
         "since": "2.2.12",
         "arity": -2,
         "container": "SLOWLOG",
-        "function": "slowlogCommand",
+        "function": "heavyLoadLogCommand",
```

Comment:
My preference was to avoid large container commands. I would rather have `slowlogCommand` call some shared function then have this call a container function.